### PR TITLE
Cloud-init default formatting fixed

### PR DESCRIPTION
**Pull request by lzap**
We render bunch of empty lines in here, this makes it better.

### CODE REVIEW

File: provisioning_templates/cloud_init/cloud_init_default.erb [hunk 1]
```diff
@@ -26,15 +26,19 @@ manage_etc_hosts: true
...
 - |
-<% if puppet_enabled %>
-<% if host_param_true?('enable-puppetlabs-repo') || host_param_true?('enable-puppetlabs-puppet6-repo') || host_param_true?('enable-puppetlabs-puppet5-repo') -%>
-<%= snippet 'puppetlabs_repo' %>
-<% end -%>
-<%= indent(2) { snippet('puppet_setup', :variables => { :full_puppet_run => true }) } %>
-<% end -%>
-
+<%=
+  if puppet_enabled && host_param_true?('enable-puppetlabs-repo') ||
```
Comment (tbrisker) [inline provisioning_templates/cloud_init/cloud_init_default.erb]: The logic here changed, you need to enclose the second condition (all the ||'s) in parenthesis to maintain same logic

Comment (lzap) [reply provisioning_templates/cloud_init/cloud_init_default.erb]: Good catch. Rebasing. If you've already branched let me know to do stable-release cherry pick PR.

Comment (tbrisker) [reply provisioning_templates/cloud_init/cloud_init_default.erb]: yes, we have branched already.

File: provisioning_templates/cloud_init/cloud_init_default.erb [hunk 2]
```diff
@@ -26,15 +26,19 @@ manage_etc_hosts: true
...
-<%= snippet 'puppetlabs_repo' %>
-<% end -%>
-<%= indent(2) { snippet('puppet_setup', :variables => { :full_puppet_run => true }) } %>
-<% end -%>
-
+<%=
+  if (puppet_enabled && host_param_true?('enable-puppetlabs-repo')) ||
+    host_param_true?('enable-puppetlabs-puppet6-repo') ||
+    host_param_true?('enable-puppetlabs-puppet5-repo')
+    indent(2) { snippet 'puppetlabs_repo' }
```
Comment (tbrisker) [inline provisioning_templates/cloud_init/cloud_init_default.erb]: still not the same logic as before. should be:
```ruby
  if puppet_enabled && (host_param_true?('enable-puppetlabs-repo') ||
    host_param_true?('enable-puppetlabs-puppet6-repo') ||
    host_param_true?('enable-puppetlabs-puppet5-repo'))

Comment (lzap) [reply provisioning_templates/cloud_init/cloud_init_default.erb]: Facepalm - I haven't looked it up.

**Review by tbrisker — APPROVED** [review]
Thanks @lzap !

**Pull request merged**